community-operators/tidb-operator/tidb-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: tidb-operator
+channels:
+    - name: beta
```

Comment:
nit: Too many spaces for this indent, please make it 2 or 0 spaces

Reply:
Fixed. PTAL